pxtrunner/tsconfig.json

```diff
@@ -15,9 +24,13 @@
             "ES2018.Promise"
         ],
         "types": [
-            "jquery",
-            "highlight.js"
```

Comment:
highlight.js uses a module system that is incompatible with the commonjs modules we use in the webapp/pxtrunner. Could probably fix this by changing the way we include highlightjs (i.e. importing it as a module instead of embedding it in the page like we do now) but I didn't want to change that as part of this PR.